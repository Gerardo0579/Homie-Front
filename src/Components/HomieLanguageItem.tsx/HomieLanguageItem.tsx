import { FC } from 'react'
import styles from './HomieLanguageItem.module.css'

interface HomieLanguageItemProps {
  Content: string
  CircleColor: string
}

export const HomieLanguageItem: FC<HomieLanguageItemProps> = ({
  Content,
  CircleColor
}) => {
  return (
    <div className={styles.LangColorContainer}>
      <div
        className={styles.LanguageCircle}
        style={{ backgroundColor: CircleColor }}
      />
      <div>{Content}</div>
    </div>
  )
}
