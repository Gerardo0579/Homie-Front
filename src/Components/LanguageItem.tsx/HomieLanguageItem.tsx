import { Space } from 'antd'
import { FC } from 'react'
import styles from './HomieLanguageItem.module.css'

interface HomieLanguageItemProps {
  Content: string
  CircleColor: string
}

export const HomieLanguageItem: FC<HomieLanguageItemProps> = ({
  Content,
  CircleColor
}) => {
  return (
    <Space>
      <div
        className={styles.LanguageCircle}
        style={{ backgroundColor: CircleColor }}
      />
      <div>{Content}</div>
    </Space>
  )
}
