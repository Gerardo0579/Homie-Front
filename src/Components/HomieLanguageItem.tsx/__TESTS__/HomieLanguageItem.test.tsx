import renderer from 'react-test-renderer'
import { HomieLanguageItem } from '../HomieLanguageItem'

const CONTENT: string = '1'
const CIRCLE_COLOR: string = '#fff'

it('renders correctly', () => {
  const tree = renderer
    .create(<HomieLanguageItem CircleColor={CONTENT} Content={CIRCLE_COLOR} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
