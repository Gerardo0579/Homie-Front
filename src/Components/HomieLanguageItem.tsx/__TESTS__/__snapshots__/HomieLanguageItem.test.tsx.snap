// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="LangColorContainer"
>
  <div
    className="LanguageCircle"
    style={
      Object {
        "backgroundColor": "1",
      }
    }
  />
  <div>
    #fff
  </div>
</div>
`;
